oracle/resources/oracle.svg

```diff
@@ -176,53 +270,154 @@
+       transform="matrix(-1.1516801,0.679836,-1.0169493,-1.7297998,202.86495,105.18681)" />
+    <path
+       sodipodi:type="arc"
+       style="fill:#ff5555;fill-opacity:1;fill-rule:nonzero;stroke:#000000;stroke-opacity:0;display:inline"
+       id="path5297-7-1"
+       sodipodi:cx="-96.635925"
+       sodipodi:cy="76.534889"
+       sodipodi:rx="2.8846545"
+       sodipodi:ry="2.8846545"
+       d="m -93.751271,76.534889 c 0,1.593151 -1.291504,2.884655 -2.884654,2.884655 -1.593151,0 -2.884655,-1.291504 -2.884655,-2.884655 0,-1.59315 1.291504,-2.884654 2.884655,-2.884654 1.59315,0 2.884654,1.291504 2.884654,2.884654 z"
+       transform="matrix(0.87199948,0,0,0.87199948,105.6435,-45.047277)" />
+    <path
+       sodipodi:type="arc"
+       style="fill:url(#linearGradient5324-8-8);fill-opacity:1;fill-rule:nonzero;stroke:#000000;stroke-opacity:0;display:inline"
+       id="path5299-6-1-49"
+       sodipodi:cx="-24.925219"
+       sodipodi:cy="38.493507"
+       sodipodi:rx="1.4573514"
+       sodipodi:ry="1.1418424"
+       d="m -23.467867,38.493507 c 0,0.630623 -0.652479,1.141843 -1.457352,1.141843 -0.804873,0 -1.457351,-0.51122 -1.457351,-1.141843 0,-0.630622 0.652478,-1.141842 1.457351,-1.141842 0.804873,0 1.457352,0.51122 1.457352,1.141842 z"
+       transform="matrix(-1.1516801,0.679836,-1.0169493,-1.7297998,31.371657,105.48325)" />
+    <path
+       sodipodi:type="arc"
+       style="fill:#2c9be5;fill-opacity:1;fill-rule:nonzero;stroke:#000000;stroke-opacity:1"
+       id="path5461"
+       sodipodi:cx="-84.135757"
+       sodipodi:cy="155.38211"
+       sodipodi:rx="69.231705"
+       sodipodi:ry="144.23273"
+       d="m -14.904053,155.38211 c 0,79.65754 -30.99609,144.23273 -69.231704,144.23273 -38.235613,0 -69.231703,-64.57519 -69.231703,-144.23273 0,-79.657535 30.99609,-144.232726 69.231703,-144.232726 38.235614,0 69.231704,64.575191 69.231704,144.232726 z"
+       transform="matrix(1.2631217,0,0,0.87199948,213.6725,14.244327)" />
+    <path
+       sodipodi:type="arc"
+       style="fill:#c87137;fill-opacity:1;fill-rule:nonzero;stroke:#000000;stroke-opacity:0"
+       id="path5461-8"
+       sodipodi:cx="-84.135757"
+       sodipodi:cy="155.38211"
+       sodipodi:rx="69.231705"
+       sodipodi:ry="144.23273"
+       d="m -14.904053,155.38211 c 0,79.65754 -30.99609,144.23273 -69.231704,144.23273 -38.235613,0 -69.231703,-64.57519 -69.231703,-144.23273 0,-79.657535 30.99609,-144.232726 69.231703,-144.232726 38.235614,0 69.231704,64.575191 69.231704,144.232726 z"
+       transform="matrix(1.228342,0,0,0.85746409,211.0465,16.713831)" />
+    <g
+       style="font-size:456.36495972000000000px;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;line-height:125%;letter-spacing:0px;word-spacing:0px;fill:#ffffff;fill-opacity:1;stroke:none;font-family:Adobe Devanagari;-inkscape-font-specification:Adobe Devanagari"
```

Comment:
Yes, and it seems to work fine, so i suppose it's just a leftover of inkspace to know what the path was created from as you suggested. :+1: